monkey/monkey_island/cc/app.py

```diff
@@ -185,6 +191,9 @@ def init_restful_endpoints(api: FlaskDIWrapper):
     api.add_resource(FileUpload)
 
     api.add_resource(PropagationCredentials)
+    api.add_resource(ConfiguredPropagationCredentials)
+    api.add_resource(StolenPropagationCredentials)
```

Comment:
Are these resources necessary? Can you just check `if request.url.endswith('/stolen-credentials')` and return different data?